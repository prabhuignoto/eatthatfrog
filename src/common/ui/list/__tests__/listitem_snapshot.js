import React from 'react';
import uuid from 'uniqid';
import ListItem from '../views/listItem';

it('render listitem correctly', () => {
  const ListItemWrapper = shallow(<ListItem
    name="ListItemTest"
    onSelect={() => {}}
    id={`listitem${uuid()}`}
    status="open"
  />);
  expect(ListItemWrapper).toMatchSnapshot();
});

