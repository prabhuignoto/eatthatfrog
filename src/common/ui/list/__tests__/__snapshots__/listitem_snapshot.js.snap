// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render listitem correctly 1`] = `
<div
  className="ui-list-item"
  key="listitem5pxulg2kjjd5bs88"
  name="list-item-listitem5pxulg2kjjd5bs88"
  role="listitem"
>
  <i
    className="open"
  />
  <button
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    ListItemTest
  </button>
</div>
`;
